text/0000-default-serializers-and-adapters.md

```diff
@@ -0,0 +1,53 @@
+- Start Date: 2019-07-27
+- Relevant Team(s): Ember Data
+- RFC PR: https://github.com/emberjs/rfcs/pull/522
+- Tracking:
+
+# Deprecate default Adapter and Serializer fallbacks
+
+## Summary
+As part of Project Trim, https://github.com/emberjs/data/issues/6166, this deprecates the fallback and default adapter and serializer across ember-data including:
+1. deprecate -default serializer fallback in Store.serializerFor
+2. deprecate Adapter.serializer and Adapter.defaultSerializer fallbacks (which currently default to -json-api.
+3. deprecate Store.defaultAdapter (-json-api) and the -json-api adapter fallback behavior
+4. deprecate Model.toJSON and support the scenario for quick debug-mode inspection via debugData() method which in DEBUG would print a similar object to the current toJSON method.
```

Comment:
I'm very much of the view we should restrict this RFC to only deprecating API and treat adding a new `debugData` function on `DS.Model` in a separate RFC.

But definitely 👍 on deprecating the fallbacks to default serializer behaviour